test/router_test.exs

```diff
@@ -197,4 +197,56 @@ defmodule Kitto.RouterTest do
     assert conn.status == 401
     Application.delete_env :kitto, :auth_token
   end
+
+  test "POST /dashboards/:id reloads single dashboard" do
+    dashboard = "sample"
+    body = %{command: "reload"}
+    conn = conn(:post, "dashboards/#{dashboard}", Poison.encode!(body))
+
+    mock = fn (topic, data) ->
+      stringified_body = for {key, val} <- data, into: %{}, do: {String.to_atom(key), val}
```

Comment:
This string atomization can be moved as `Kitto.TestHelper.atomize_map` in `test/test_helper.exs`